src/stockwatch/use_cases/importing.py

```diff
@@ -122,10 +138,29 @@ def _process_valuta_exchange_row(row: dict[str, str]) -> CurrencyExchange:
     return CurrencyExchange(exchange_datetime, rate, amount_from)
 
 
+def _process_cash_settlement_row(
+    row: dict[str, str],
+    exchanges: list[CurrencyExchange],
+) -> CashSettlement:
+    date_time_str = row["Datum"] + ";" + row["Tijd"]
+    settlement_datetime = datetime.strptime(date_time_str, "%d-%m-%Y;%H:%M")
+    isin = IsinStr(row["ISIN"])
+    curr = row["Mutatie"]
+    amount = Amount(float(row["Bedrag"].replace(",", ".")), curr)
+    if curr != "EUR":
+        amount = apply_exchange(
```

Comment:
I'm not really sure if we should apply the exchange rate here. It feels like a bit of a dirty trick to ensure that `amount == 0.0` in the process sell transaction.

Reply:
No, not related to that, see my other comment. The logic is that as soon as I find a transaction in non-EUR, I apply an exchange. And for cash settlements, there is no need to retain non-EUR amounts I think.

I prefer to keep the logic as proposed here, can you agree?

Reply:
I have taken the whole delisting processing out of this method, much clearer now I think.